po/de.po

```diff
@@ -8660,7 +8599,7 @@ msgstr "Dieser Korb ist öffentlich zugänglich unter folgender Adresse:"
 #: modules/webbasket/lib/webbasket_templates.py:2488
 #, fuzzy
 msgid "This basket does not contain any records yet."
-msgstr "Diese Sammlung enthält zur Zeit keine Dokumente."
+msgstr "Dieser Korb enthält zur Zeit keine Dokumente."
```

Comment:
If you review and fix message translations, then the `fuzzy` directive has to be removed, otherwise the translations wouldn't be taken into account.

Instead of amending this in this PR, we have moved to Transifex also for the Invenio legacy translations, and there have been quite a few updates since then. Can you please look there and amend the translations there directly?

- https://www.transifex.com/inveniosoftware/invenio/invenio-maint-11-messages/

Many thanks and sorry for not integrating this PR before moving to Transifex.